assets/js/components/Project/Form.jsx

```diff
@@ -31,12 +31,41 @@ const ProjectForm = ({ showForm, setShowForm }) => {
     }
   );
 
+  let [
+    updateProject,
+    {
+      loading: projectUpdateLoading,
+      error: projectUpdateError,
+      projectUpdateData,
```

Comment:
I'd say if we're not using these values, let's not include the object argument and keep it cleaner.